RedHatInsights/compliance-frontend
# feat(RHINENG-11721): Migrate the edit policy modal

Implements https://issues.redhat.com/browse/RHINENG-11721.

Migrates the edit policy wizard to REST. 

Remaining bugs:

- [ ] When deselecting the last system with particular OS minor version, the rules tab should hide this OS minor version tab and not display the rules for this OS minor version.
- [ ] When in the rules tab, selecting all rules makes the table load selection infinitely.
- [x] ~~When adding new system that has OS minor version different to those already associated with the policy, after clicking submit, the modal errors (it assigns the systems, but there is a problem finding the right newly created tailoring ID that we can use to assign new rules).~~

## How to test

Navigate to any policy or open the policies list. Click Edit rules/systems for this policy and wait for the modal to open and load tabs. Edit rules or systems. After submit, verify that the changes are applied to this policy

## Blocked by

~~https://github.com/RedHatInsights/compliance-frontend/pull/2232~~
~~https://github.com/RedHatInsights/compliance-frontend/pull/2270~~
